deployment/docker-compose.template.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  api:
+    image: ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com/peerbook/server:${VERSION}
+    command: sh -c "/bin/chamber exec ${LAYER_NAME} -- /bin/myapp"
```

Comment:
Move to dockerfile.prod. 
Also, I think it's `/app/server` and not `/bin/myapp`.